packages/shared-core/src/filters.ts

```diff
@@ -204,12 +235,29 @@ export const buildLuceneQuery = filter => {
   return query
 }
 
+const deepGet = (obj: { [x: string]: any }, key: string) => {
+  if (!obj || !key) {
+    return null
+  }
+  if (Object.prototype.hasOwnProperty.call(obj, key)) {
+    return obj[key]
+  }
+  const split = key.split(".")
+  for (let i = 0; i < split.length; i++) {
+    obj = obj?.[split[i]]
+  }
+  return obj
+}
+
 /**
  * Performs a client-side lucene search on an array of data
  * @param docs the data
  * @param query the JSON lucene query
  */
-export const runLuceneQuery = (docs, query) => {
+export const runLuceneQuery = (
+  docs: any[],
+  query?: { [x: string]: any; sheet?: string }
```

Comment:
NAB: Seems a little strange to me to have a property called `sheet` in a method as generic as this.

Reply:
This was autogenerated, not sure why? 😅